docs/websites/varnish/use-varnish-with-nginx-to-serve-cached-wordpress-content-over-ssl-on-debian-8.md

```diff
@@ -0,0 +1,683 @@
+
+For HTTPS traffic, nginx will listen on port 443 and send decrypted traffic to Varnish on port 80. If content is found in cache, Varnish will send the unencrypted content from cache back to nginx, which will encrypt it and send it to the client. If content is not found in cache Varnish will request it from backend nginx on port 8080, store it in cache and then send it unencrypted to frontend nginx which will encrypt it and send it to the client's browser.
+
+Our setup is illustrated below. Please note that frontend nginx and backend nginx are one and the same server:
+
+[![Nginx-Varnish-Nginx server configuration diagram](/docs/assets/varnish-scheme-small.png)](/docs/assets/varnish-scheme.png)
+
+## Before You Begin
+
+This tutorial assumes that you have SSH access to your Linode running Debian 8 (Jessie). Before you get started, make sure you've completed the following:
+
+*   Complete the steps in our [Getting Started](https://linode.com/docs/getting-started) and [Securing your Server](https://www.linode.com/docs/security/securing-your-server) guides.
+
+*   Follow the steps outlined in our [LEMP on Debian 8](https://www.linode.com/docs/websites/lemp/lemp-server-on-debian-8). You should skip the nginx configuration section as we'll address it later on in our tutorial.
+
+*   Follow the steps in our [WordPress](https://www.linode.com/docs/websites/cms/how-to-install-and-configure-wordpress) guide to install and configure WordPress.
+
+## Install and Configure Varnish
+
+1.  Issue the following commands to install Varnish:
+
+		sudo apt-get update
+		sudo apt-get install varnish
+
+2.  Use 'nano' or other text editor to edit the `/etc/default/varnish` file:
+
+		sudo nano /etc/default/varnish
+
+3.  To make Varnish start at boot, under `Should we start varnishd at boot?` set the `START` parameter to `yes`:
+
+		START=yes
+
+4. In the `Alternative 2` section, make the following changes to the `DAEMON_OPTS` section:
+
+	{: .file-excerpt }
+	/etc/default/varnish
+	:   ~~~ conf
+	
+		DAEMON_OPTS="-a :80 \
+				 	-T localhost:6082 \
+				 	-f /etc/varnish/custom.vcl \
+				 	-S /etc/varnish/secret \
+				 	-s malloc,500M"
+		~~~
+
+	This will set Varnish to listen on port 80, and will instruct it to use the `custom.vcl` configuration file. If you make the configuration changes in the `default.vcl` file, a future update to Varnish may overwrite it. Save and exit the file.
+
+	The `-s malloc,500M` line sets the maximum amount of RAM that will be used by Varnish to store content. This value can be adjusted to suit your needs, taking into account the server's total RAM along with the size and expected traffic of your website. For example, on a system with 4 GB of RAM , you can allocate 1 or 2 or 3 GB to Varnish but obviously not all the system's RAM. To allocate 1 GB you should write: `-s malloc,1G`.
```

Comment:
As with line 83, is there a difference between `g` and `G` as far as Varnish is concerned?